10K_Risk_Factors Deliverable 1

e-south opened the pull request:
## Description of Change

Adding html_parser, initial cluster framework, etc. See more details in deliverable_1.md.

## Team Name and Contributers

10K_Risk_Factors-- Eric South, Nicholas Mosca, Evie Wan

## Checklist

Before creating your PR, please ensure the following requirements.

### General

- [ ] PR does **not** include irrelevant or hidden files (i.e. `__pycache__`, `.vscode`... etc.)
- [ ] Changes are in the correct folder
- [ ] There are no merge conflicts
- [ ] Commit messages are sensible and descriptive

### Code

- [ ] Implementation matches proposed design
- [ ] Documentation on how to run the code is created/updated
- [ ] Code is legible

### Report

- [ ] Report meets the deliverable criteria
- [ ] Proposal is up to date with latest decisions

e-south committed:
Updating repo structure and adding html_parser.py

e-south committed:
Updating repo structure and adding html_parser

njmosca committed:
first edits

njmosca committed:
editing and adding path_mover.py

eviewan committed:
adding financial metrics parsing file

eviewan committed:
adding scrum file

e-south committed:
Merge pull request #2 from e-south/data_gathering

Data gathering

e-south committed:
Merge pull request #3 from ewan212/master

adding financial metrics parsing file

e-south committed:
adding deliverable 1 and gitignore

e-south committed:
Delete .ipynb_checkpoints directory

e-south committed:
Delete .DS_Store

gallettilance reviewed (changes requested):
lgtm but please remove all the .DS_Store files

e-south committed:
Delete .DS_Store

e-south committed:
Delete .DS_Store

e-south committed:
Delete .DS_Store

e-south commented:
> lgtm but please remove all the .DS_Store files

thanks for catching that-- I've deleted .DS_Store files. I've also added a gitignore file so hopefully this issue doesn't repeat.

e-south committed:
updating html_parser and adding topic models

njmosca committed:
Merge pull request #4 from e-south/topic_models

updating html_parser and adding topic models
